### PR TITLE
Add Slack to `README.md` and documentation site 🗨️ 

### PR DESCRIPTION
**Pull request by maoo**
Note! we should probably add it to the website too!

### CODE REVIEW

**Comment by maoo** [comment]
@JamieSlome - done; I used https://simpleicons.org/?q=slack and https://www.urlencoder.org/ ; for some reason, the white version (for dark mode) is the same as the black one; not sure if I'm missing something stupid here.

File: README.md [hunk 1]
```diff
@@ -106,4 +106,6 @@ This project is distributed under the Apache-2.0 license. See <a href="./LICENSE
 
 ## Contact
 
+Join the [Git Proxy Slack Channel on finos-lf.slack.com](https://app.slack.com/client/T01E7QRQH97/C06LXNW0W76).
+
```
Comment (JamieSlome) [inline README.md:110]: Can we do a Slack badge as well at the top?

Comment (maoo) [reply README.md:110]: done!

**Comment by JamieSlome** [comment]
Seems like the deployment failed 🤔 

To the side of that, just open up an SVG viewer. Does the two images appear as expected?

**Comment by maoo** [comment]
> Seems like the deployment failed 🤔

I kicked it manually and it worked - https://deploy-preview-456--endearing-brigadeiros-63f9d0.netlify.app/
 
> To the side of that, just open up an SVG viewer. Does the two images appear as expected?

Nope, the white version shows as black also on my local env; I'm missing something on how to use https://simpleicons.org/?q=slack download feature 😄

**Pull request merged**